packages/client/hmi-client/src/page/Home.vue

```diff
@@ -218,7 +218,12 @@ onMounted(async () => {
 	resourcesStore.reset(); // Project related resources saved.
 	queryStore.reset(); // Facets queries.
 
-	projects.value = ((await ProjectService.home()) ?? []).slice().reverse();
+	projects.value = ((await ProjectService.getAll()) ?? []).slice().reverse();
+
+	projects.value.forEach(async (project) => {
+		project.assets = await ProjectService.getAssets(project.id);
+		project.relatedDocuments = await getRelatedDocuments(project.id, 'xdd-covid-19');
```

Comment:
Could you hardcode this `xdd-covid=19` within the `getRelatedDocuments()` function directly?

Reply:
yup

Reply:
just a heads up here @Jami159